awx/api/views/__init__.py

```diff
@@ -4350,3 +4351,17 @@ def post(self, request, *args, **kwargs):
             return Response({"error": _("This workflow step has already been approved or denied.")}, status=status.HTTP_400_BAD_REQUEST)
         obj.deny(request)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class PeersList(ListAPIView):
+    name = _("Peers")
+    model = models.InstanceLink
+    serializer_class = serializers.InstanceLinkSerializer
+    search_fields = ('source', 'target', 'link_state')
+
+
+class PeersDetail(RetrieveAPIView):
```

Comment:
The plural of "Peers" is kind of weird here - this is showing a single InstanceLink right?

Reply:
I agree with @AlanCoding (with the additional question in my mind of why they exist)